include/clap/helpers/plugin.hxx

```diff
@@ -791,6 +800,69 @@ namespace clap { namespace helpers {
       return self.paramsCount();
    }
 
+   //--------------------------------------//
+   // clap_plugin_configurable_audio_ports //
+   //--------------------------------------//
+
+   template <MisbehaviourHandler h, CheckingLevel l>
+   bool Plugin<h, l>::clapConfigurableAudioPortsCanApplyConfiguration(
+      const clap_plugin_t *plugin,
+      const clap_audio_port_configuration_request *requests,
+      uint32_t request_count) noexcept {
```

Comment:
Was this formatted using the .clang-format in the repo?
Maybe add a line break so we can distinguish the arguments and function body.

Reply:
If it was formatted with our .clang-format then ignore my comment 👍

Reply:
Just reformatted the section again with VS2022 which claims having detected the project's clang-tidy and now diff showed up in git.